---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Main Work Interests:
======

* Single-Cell RNA Sequencing data analysis on Uveal Melanoma. Working on Intra-tumoral cells heterogeneity to define new markers specific to Progression/Invasion and Metastatic signatures. A better understanding of this heterogeneity will lead to better diagnostics and treatments.
* RNAseq analysis on DLBCL cohort. Identification of new markers to diagnose efficiently patients with response to monoclonal antibody treatment (Rituxan)+CHOP.
* Exome-Wide Association Studies on germline mutations modulates response to anti-PD1 and anti-CTLA4 immunotherapy in melanoma patients.
* Development of tools for integrative analysis of DNA-seq, RNA-seq, Chip-Seq and Whole-Exome Sequencing datasets in the context of cancer genomics.

Education
======
* Postgraduate degree in Bioinformatics, University of Mediterranean Luminy, Marseilles, France, 2005
* M.Sc in Computer Sciences, University of Mediterranean Luminy, Marseilles, France, 2004
* B.S. in Mathematics & Computer Science, University of Mediterranean Luminy, Marseilles, France, 2003

Work experience
======
* June 2016 - December 2017: Engineer Bioinformatician
  * INSERM UMR_S-1065, Centre Méditerranéen de Médecine Moléculaire, Nice, France.
  * Single Cell RNAseq study on uveal melanoma: Identification of molecular pathways triggered by gene expression on heterogeneous cells in tumors.
  * Bioinformatics analysis for projects in the network of Cancéropôle PACA.
  * Strong interaction with scientists. Definition of analysis pipeline according to their requirements and objectives, principally on cancer genomics.
  * Public data analysis (GEO, TCGA), meta-analysis pipeline in R.
  * Supervisor: Bioinformatic Master Student, 6 months.

* January 2013 - March 2016: Research Scientist
  * Genome Sciences, University of Washington, Seattle, USA.
  * Member of the University of Washington Center for Mendelian Genomics Consortium (UW-CMG).
  * Study of Mendelian diseases using WES and WGS data

* January 2009 – December 2012: Engineer Bioinformatician
  * CRS4, Cagliari, Italy.
  * 1000 Sardinian Genomes, NGS data analysis.

* May 2006 – January 2009: Engineer Bioinformatician
  * CRS4, Cagliari, Italy.
  * Development: Data warehouse and platform analysis of microarrays for Cancer specific study.
  * Web development on Zope Platform.

Skills
======
* Operating System: Linux, MacOS. Strong Experience on Linux administration
* Programming Language: Perl, Python, C, R/Bioconductor.
* Bioinformatics Applications:
  * Samtools, Picard, GATK, BWA, Bowtie, MEME, Tophat, Cufflinks, DESeq, HTseq, Bedtools, CoNIFER, XHMM.
* Pathways Analysis: GSEA, IPA.

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
