---
title: "Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel"
collection: publications
permalink: /publication/2015-02-06-Delaneau-O
excerpt: 'A major use of the 1000 Genomes Project (1000 GP) data is genotype imputation in genome-wide association studies (GWAS). Here we develop a method to estimate haplotypes from low-coverage sequencing data that can take advantage of single-nucleotide polymorphism (SNP) microarray genotypes on the same samples. First the SNP array data are phased to build a backbone (or &apos;scaffold&apos;) of haplotypes across each chromosome. We then phase the sequence data &apos;onto&apos; this haplotype scaffold. This approach can take advantage of relatedness between sequenced and non-sequenced samples to improve accuracy. We use this method to create a new 1000 GP haplotype reference set for use by the human genetic community. Using a set of validation genotypes at SNP and bi-allelic indels we show that these haplotypes have lower genotype discordance and improved imputation performance into downstream GWAS samples, especially at low-frequency variants.'
date: 2015-02-06
venue: 'Nat Commun.'
paperurl: 'http://fred3ric.github.io/files/2015-02-06-Delaneau-O.pdf'
citation: 'Delaneau O, Marchini J; <b>1000 Genomes Project Consortium.</b>; 1000 Genomes Project Consortium'
---

<a href='http://fred3ric.github.io/files/2015-02-06-Delaneau-O.pdf'>Download paper here</a>

A major use of the 1000 Genomes Project (1000 GP) data is genotype imputation in genome-wide association studies (GWAS). Here we develop a method to estimate haplotypes from low-coverage sequencing data that can take advantage of single-nucleotide polymorphism (SNP) microarray genotypes on the same samples. First the SNP array data are phased to build a backbone (or &apos;scaffold&apos;) of haplotypes across each chromosome. We then phase the sequence data &apos;onto&apos; this haplotype scaffold. This approach can take advantage of relatedness between sequenced and non-sequenced samples to improve accuracy. We use this method to create a new 1000 GP haplotype reference set for use by the human genetic community. Using a set of validation genotypes at SNP and bi-allelic indels we show that these haplotypes have lower genotype discordance and improved imputation performance into downstream GWAS samples, especially at low-frequency variants.

Recommended citation: Delaneau O, Marchini J; 1000 Genomes Project Consortium.; 1000 Genomes Project Consortium
